Fastfile.private

```diff
@@ -138,12 +137,15 @@ reflected in `Info.plist` during building."
     end
   end
 
-  desc "Executes the tests for the project using `scan`. This lane uses the configuration mapped to `:test`."
-  private_lane :run_tests do
+  desc "Executes the tests for the project using `scan`."
+  private_lane :run_tests do |options|
+
+    build_configuration_key = options[:build_configuration]
```

Comment:
Could there be tests and would we want to run them in something that's not development?

Reply:
I don't think we want to do this, but fastlane supports it.